extras/jsonrpc/daemon.go

```diff
@@ -178,9 +182,21 @@ func (d *Client) AccountFund(fromAccount string, toAccount string, amount string
 
 func (d *Client) AccountCreate(accountName string, singleKey bool) (*AccountCreateResponse, error) {
 	response := new(AccountCreateResponse)
-	return response, d.call(response, "account_create", map[string]interface{}{
-		"account_name": accountName,
-		"single_key":   singleKey,
+	args := struct {
+		AccountName string `json:"account_name"`
+		SingleKey   bool   `json:"single_key"`
+	}{
+		AccountName: accountName,
+		SingleKey:   singleKey,
+	}
+	structs.DefaultTagName = "json"
+	return response, d.call(response, "account_create", structs.Map(args))
```

Comment:
this is rather useless because you're going from a struct to a map through a serializer when in fact you can just define the map on the go:

```
map[string]interface{}{
		"account_name": accountName,
    "single_key": singleKey,
	})
```

Reply:
I agree. was trying to fix something entirely unrelated